gateway-ha/pom.xml

```diff
@@ -128,6 +128,16 @@
         <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>http-client</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.eclipse.jetty.http2</groupId>
+                    <artifactId>jetty-http2-common</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.eclipse.jetty.http2</groupId>
+                    <artifactId>jetty-http2-server</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Please explain why this exclusion is safe. We recently broke this project by dependency exclusion https://github.com/trinodb/trino-gateway/pull/709.